app/assets/stylesheets/application.scss

```diff
@@ -10,8 +10,7 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
- *= require_self
- *= require foundation_and_overrides
 
  */
+@import "mystyles.scss";
+@import "./angled-edges-master/assets/styles/angled-edges.scss"
```

Comment:
heaven help me. 

Ill allow line 15 to be remove, but you need to add lines 13 and 14 back